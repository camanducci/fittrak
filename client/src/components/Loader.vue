<template>
<v-layout justify-center align-center>
  <v-progress-circular 
    color="cyan"
    indeterminate 
    size=128
  >
  </v-progress-circular>
</v-layout>
</template>

<script>

export default {
  mounted: function () {
    // Fresh data from the API
    // For now we'll mock this with a setTimeout
    setTimeout(() => {
      this.$store.dispatch('refreshData', {
        meta: {}
      })
    }, 1500)
  }
}
</script>
