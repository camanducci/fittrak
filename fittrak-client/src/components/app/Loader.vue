<template>
  <v-progress-circular v-if="true == false" color="white" size="24" :indeterminate="true"></v-progress-circular>
</template>

<script>
export default {};
</script>
