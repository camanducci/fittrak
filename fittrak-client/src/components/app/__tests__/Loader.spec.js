import { shallowMount } from "@vue/test-utils";

import Loader from "@/components/app/Loader";

describe("Loader", () => {
  test("Loader renders", () => {
    const component = shallowMount(Loader, {
      propsData: {
        color: "red",
        size: "48"
      }
    });
    expect(component.html()).toMatchSnapshot();
  });
});
