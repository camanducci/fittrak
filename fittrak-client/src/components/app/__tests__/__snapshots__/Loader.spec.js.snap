// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Loader renders 1`] = `
<v-flex-stub tag="div" text-xs-center="">
  <v-progress-circular-stub indeterminate="true" rotate="0" size="48" width="4" value="0" color="red"></v-progress-circular-stub>
</v-flex-stub>
`;
